aiohttp/client_reqrep.py

```diff
@@ -39,12 +39,18 @@
 __all__ = ('ClientRequest', 'ClientResponse', 'RequestInfo', 'Fingerprint')
 
 
-ContentDisposition = collections.namedtuple(
-    'ContentDisposition', ('type', 'parameters', 'filename'))
+@attr.s(frozen=True)
```

Comment:
I have my doubts about the usage of `frozen=True` taking into account that immutability depends on the type of fields that you have in the data class. In the worst case scenario having a mutable attribute, the `hash` will fail and you won't prevent modify these attributes.

Reply:
Sorry, I don't follow your point.

`ConetentDisposition` was immutable named tuple, I've just converted the class into immutable `attrs` representation. All `type`, `parameters` and `filename` attributes are immutable too.

Reply:
```>>> @attr.s(frozen=True)
... class Foo:
...     a = attr.ib(type=MappingProxyType)
...
>>> d = {}
>>> m = MappingProxyType(d)
>>> f = Foo(m)
>>> hash(f)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<attrs generated hash a57df858a086a8e0acf97d938d787d39b6f33d51>", line 4, in __hash__
TypeError: unhashable type: 'mappingproxy'```